### PR TITLE
FIX: In topic search for glimmer header

### PR DESCRIPTION
**Pull request by janzenisaac**

### CODE REVIEW

**Review by CvX — APPROVED** [review]

**Comment by tgxworld** [comment]
Is there a test we can add for this change here?

**Comment by janzenisaac** [comment]
> Is there a test we can add for this change here?

We are going to be creating over 20 topics to test, I don’t think this is worth the test.

**Comment by tgxworld** [comment]
> We are going to be creating over 20 topics to test, I don’t think this is worth the test.

Can we stub the number of topics before the in-topic search is triggered? Personally, I think this feature is important enough to warrant a test here to help us catch regression in the future.

File: spec/system/header_spec.rb [hunk 1]
```diff
@@ -189,23 +196,37 @@
...
+        header.search_in_topic_keyboard_shortcut
+        expect(search).to have_search_menu_visible
+      end
+
+      it "closes search after two presses" do
+        visit "/t/#{topic.slug}/#{topic.id}"
+        header.search_in_topic_keyboard_shortcut
+        header.search_in_topic_keyboard_shortcut
+        expect(search).not_to have_search_menu_visible
+      end
```
Comment (davidtaylorhq) [inline spec/system/header_spec.rb:229]: Booting the app in system specs is quite costly, so IMO we should combine these two specs. Something like

```
it "opens search on first ctrl+f, then closes on second"
```

**Review by davidtaylorhq — APPROVED** [review]

**Pull request merged**